pkg/webservice/handlers.go

```diff
@@ -240,6 +240,7 @@ func getApplicationJSON(app *objects.Application) *dao.ApplicationDAOInfo {
 			Count:         taskGroup.Count,
 			MinResource:   taskGroup.MinResource.DAOMap(),
 			Replaced:      taskGroup.Replaced,
+			Timeout:       taskGroup.Timedout,
```

Comment:
Missing "d" in the name: Time**d**Out Also capitalise the `O`

Reply:
Done.Thanks.